pom.xml

```diff
@@ -271,7 +271,7 @@
             <plugin>
                 <groupId>org.itsallcode</groupId>
                 <artifactId>openfasttrace-maven-plugin</artifactId>
-                <version>0.1.0</version>
+                <version>1.0.0</version>
```

Comment:
Why did you change a major version? This branch looks as 0.2.0 to me

Reply:
It's just the version of the openfasttrace-maven-plugin. I did not change ther version number yet.